README.md

```diff
@@ -1,106 +1,170 @@
+- `CONJUR_ACCOUNT` - Conjur account that you are connecting to. This value is set during Conjur deployment.
+- `CONJUR_AUTHN_LOGIN` - User/host identity
+- `CONJUR_AUTHN_API_KEY` - User/host API key
+- `CONJUR_AUTHN_URL` - (optional) Alternate authn endpoint. Default endpoint uses `<applianceUrl>/authn`
+  for generic username and password/API key login flow.
 
-For example, specify the environment variables like so:
+For example, you can specify the environment variables like so:
 
-```bash
-CONJUR_ACCOUNT=myorg
-CONJUR_AUTHN_LOGIN=host/myhost.example.com
-CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
-CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api
+```sh-session
+export CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api
+export CONJUR_ACCOUNT=myorg
+export CONJUR_AUTHN_LOGIN=host/myhost.example.com
+export CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
 ```
 
-In order to set `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY`, you will need to add the host to Conjur with the proper privileges in policy. 
+or you could provide these at runtime to your jar:
+```sh-session
+$ CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api \
+  CONJUR_ACCOUNT=myorg \
+  CONJUR_AUTHN_LOGIN=host/myhost.example.com \
+  CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s \
+  java -jar myConjurClient.jar
+```
+If you are using a host-based user like this example shows, you will need to add the host
+to Conjur with the proper privileges in policy in order to know the appropriate
+`CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY` values.
 
-For this _specific_ example, we will define the Java app as a host and grant permissions over some variable. This variable will later hold a secret and we will 
-show how to retrieve this secret using the Conjur Java API once a connection is established. If you have a different use case, you can use the below policy as the basis for 
-another desired outcome. If a policy already exists for your host, you can skip the next set of steps.
+For this _specific_ example, we will define the Java app as a host and grant permissions over
+some variable. This variable will later hold a secret and we will show how to retrieve this
+secret using the Conjur Java API once a connection is established. If you have a different use
+case, you can use the below policy as the basis for another desired outcome. If a policy
+already exists for your host, you can skip the next set of steps.
 
 1. Make sure Conjur is running in the background and open the Conjur CLI.
-    
-    _NOTE:_ If using Docker: `docker-compose exec <NAME_OF_CLI_CONTAINER> bash`
 
-2. Copy the following policy, substitute: the name of the policy, hostname, and permissions accordingly, and save it as a `.yml` file.
+   _NOTE:_ If using Docker: `docker-compose exec <NAME_OF_CLI_CONTAINER> bash`
 
-```yaml
+2. Copy the following policy and substitute `<>`-enclosed variables accordingly, and
+   save it as a `.yml` file.
+   ```yaml
 - !policy
```

Comment:
the content in the code block needs to be indented to render properly

Reply:
No longer relevant - example scrapped